src/libs/ajax.js

```diff
@@ -1667,7 +1668,11 @@ const Disks = signal => ({
       details: async () => {
         const res = await fetchLeo(`api/google/v1/disks/${project}/${name}`,
           _.mergeAll([authOpts(), appIdentifier, { signal, method: 'GET' }]))
-        return res.json()
+        const resJs = res.json().then(val => {
+          val.diskType = pdTypes.fromString(val.diskType)
+          return val
+        })
+        return resJs
```

Comment:
Would the following be more concise and consistent with how we typically mutate objects (using `_.set()`)?

```suggestion
        return res.json().then(val => _.set('diskType', pdTypes.fromString(val.diskType), val))
```

Reply:
Thank you for this! This looks way better